raster3d/r3.mkdspf/r3.mkdspf.html

```diff
@@ -61,11 +61,5 @@ <h2>NOTE</h2>
 copy them to their mapset.  
 
 <h2>AUTHOR</h2>
-<p>Bill Brown,
-<a href="mailto:brown@gis.uiuc.edu">bbrown@gis.uiuc.edu</a>
 
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
+Bill Brown, <a href="mailto:brown@gis.uiuc.edu">bbrown@gis.uiuc.edu</a>
```

Comment:
`<h2>Example:</h2>` header should be `<h2>EXAMPLE:</h2>`

Reply:
fixed